src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -6,29 +6,52 @@
 using System.Threading;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Microsoft.CodeAnalysis.CSharp.Symbols;
-using Microsoft.CodeAnalysis.Syntax;
 using Microsoft.CodeAnalysis.Operations;
+using Microsoft.CodeAnalysis.Syntax;
 
 namespace Microsoft.CodeAnalysis
 {
     public static class CSharpExtensions
     {
+        /// <summary>
+        /// Determines if <see cref="SyntaxToken"/> is of a specified kind.
+        /// </summary>
+        /// <param name="token">The source token.</param>
+        /// <param name="kind">The syntax kind to test for.</param>
+        /// <returns><c>true</c> if token is of specified kind; <c>false</c> otherwise.</returns>
         public static bool IsKind(this SyntaxToken token, SyntaxKind kind)
         {
             return token.RawKind == (int)kind;
         }
 
+        /// <summary>
+        /// Determines if <see cref="SyntaxTrivia"/> is of a specified kind.
+        /// </summary>
+        /// <param name="trivia">The source trivia.</param>
+        /// <param name="kind">The syntax kind to test for.</param>
+        /// <returns><c>true</c> if trivia is of specified kind; <c>false</c> otherwise.</returns>
         public static bool IsKind(this SyntaxTrivia trivia, SyntaxKind kind)
         {
             return trivia.RawKind == (int)kind;
         }
 
+        /// <summary>
+        /// Determines if <see cref="SyntaxNode"/> is of a specified kind.
+        /// </summary>
+        /// <param name="node">The source node.</param>
+        /// <param name="kind">The syntax kind to test for.</param>
+        /// <returns><c>true</c> if node is of specified kind; <c>false</c> otherwise.</returns>
         public static bool IsKind(this SyntaxNode node, SyntaxKind kind)
         {
             return node?.RawKind == (int)kind;
         }
 
+        /// <summary>
+        /// Determines if <see cref="SyntaxNodeOrToken"/> is of a specified kind.
+        /// </summary>
+        /// <param name="nodeOrToken">The source node or token.</param>
+        /// <param name="kind">The syntax kind to test for.</param>
+        /// <returns><c>true</c> if node or token is of specified kind; <c>false</c> otherwise.</returns>
```

Comment:
```suggestion
        /// <returns><see langword="true"/> if the node or token is of the specified kind; otherwise, <see langword="false"/>.</returns>
```